tests/scripts/setup

```diff
@@ -102,8 +119,11 @@ def node_setup(quest)
     create_node(opts)
   end
   update_docker_hosts
-  run_puppet_on_nodes if run_puppet_after
-  wait_for_ssh
+  poll_nodes("SSH", 10){ |name| query_ssh(name) }
```

Comment:
Does SSH running actually indicate that Puppet is ready to run?

Reply:
@binford2k If there isn't a puppet run taking place during the setup process, puppet should be ready to run as soon as the container is running. Here, availability of the SSH service is what determines when the container is ready for a user to connect. In cases when there is a puppet run, the conditional right after this line handles waiting for that to complete.

Reply:
👍